src/libraries/System.Net.Sockets/src/System/Net/Sockets/TCPListener.cs

```diff
@@ -159,14 +160,22 @@ public void Start(int backlog)
             _active = true;
         }
 
-        // Closes the network connection.
+        /// <summary>
+        /// Closes the network connection.
+        /// </summary>
         public void Stop()
         {
             _serverSocket?.Dispose();
             _active = false;
             _serverSocket = null;
         }
 
+        /// <summary>
+        /// Releases all resources used by the current <see cref="TcpListener"/> instance.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
The API proposal doesn't recommend to make it `EditorBrowsableState.Never`, and I don't think there is an advantage hiding `Dispose`, so I would remove this.